packages/lib/pod/src/podChecks.ts

```diff
@@ -0,0 +1,340 @@
+// TODO(POD-P3): Decide if these utils should all be published outside
```

Comment:
Given your pivot to splitting out serialization instead, should this file be removed?